The latest code exit abnormally when using  cuckatoo with wrong edge_bits

```
$ ./qitmeer-miner -C qitmeer.too.conf
2019-11-23|10:55:19.651 [INFO ] Found Device                        module=miner platform=Apple minerID=0 deviceName="Intel(R) HD Graphics 630" MaxWorkGroupSize(MB)=256 MaxMemAllocSize(MB)=384.000
2019-11-23|10:55:19.651 [INFO ] Found Device                        module=miner platform=Apple minerID=1 deviceName="AMD Radeon Pro 560 Compute Engine" MaxWorkGroupSize(MB)=256 MaxMemAllocSize(MB)=1024.000
2019-11-23|10:55:19.651 [INFO ] solo miner start                    module=miner
2019-11-23|10:55:19.653 [ERROR] cuckatoo edge_bits cannot less than 29! module=miner
2019-11-23|10:55:19.654 [ERROR] cuckatoo edge_bits cannot less than 29! module=miner
2019-11-23|10:55:19.654 [INFO ] listen new work server              module=miner
2019-11-23|10:55:19.654 [INFO ] listen submit block server          module=miner
2019-11-23|10:55:25.760 [INFO ] getBlockTemplate height:25638 , target :min difficulty 1856 module=miner
panic: runtime error: invalid memory address or nil pointer dereference
        panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x45323e2]

goroutine 23 [running]:
github.com/Qitmeer/go-opencl/cl.releaseProgram(0x0)
        /work/go/pkg/mod/github.com/!qitmeer/go-opencl@v0.0.0-20190704222003-c93200893312/cl/program.go:36 +0x22
github.com/Qitmeer/go-opencl/cl.(*Program).Release(...)
        /work/go/pkg/mod/github.com/!qitmeer/go-opencl@v0.0.0-20190704222003-c93200893312/cl/program.go:43
qitmeer-miner/symbols/qitmeer.(*Cuckatoo).Release(0xc00011c600)
        /work/qitmeer/qitmeer-miner/symbols/qitmeer/pow_cuckatoo.go:288 +0x48
panic(0x467d9a0, 0x4ccd1e0)
        /usr/local/go/src/runtime/panic.go:679 +0x1b2
github.com/Qitmeer/go-opencl/cl.(*CommandQueue).EnqueueFillBuffer.func1(0xc000130210, 0x0, 0xc0000e502c, 0x4, 0x0, 0x4000000, 0x0, 0x0, 0x0, 0xc0000101f0, ...)
        /work/go/pkg/mod/github.com/!qitmeer/go-opencl@v0.0.0-20190704222003-c93200893312/cl/queue.go:189 +0x36
github.com/Qitmeer/go-opencl/cl.(*CommandQueue).EnqueueFillBuffer(0xc000130210, 0x0, 0xc0000e502c, 0x4, 0x0, 0x4000000, 0x0, 0x0, 0x0, 0x400eca8, ...)
        /work/go/pkg/mod/github.com/!qitmeer/go-opencl@v0.0.0-20190704222003-c93200893312/cl/queue.go:189 +0xca
qitmeer-miner/symbols/qitmeer.(*Cuckatoo).InitParamData(0xc00011c600)
        /work/qitmeer/qitmeer-miner/symbols/qitmeer/pow_cuckatoo.go:299 +0x11a
qitmeer-miner/symbols/qitmeer.(*Cuckatoo).Mine(0xc00011c600, 0xc000027f40)
        /work/qitmeer/qitmeer-miner/symbols/qitmeer/pow_cuckatoo.go:160 +0x98b
created by qitmeer-miner/symbols/qitmeer.(*QitmeerRobot).Run
        /work/qitmeer/qitmeer-miner/symbols/qitmeer/robot.go:103 +0x2a6
```

--- a/symbols/qitmeer/pow_cuckatoo.go
+++ b/symbols/qitmeer/pow_cuckatoo.go
@@ -74,6 +74,7 @@ func (this *Cuckatoo) InitDevice() {
 	}
 	if this.Cfg.OptionConfig.EdgeBits < 29{
 		common.MinerLoger.Error("cuckatoo edge_bits cannot less than 29!")
+		this.IsValid = false
 		return
 	}
 	edges_bits = uint(this.Cfg.OptionConfig.EdgeBits)

